### DISCUSSION TITLE
[Feature request] Drag and drop notes into a folder/tag to add the folder/tag

### DISCUSSION THREAD

**Discussion by makarandh** [standardnotes/forum#2516] [Suggestions]
I am unable to move notes from one folder to another. When I right click on notes or folders there is no context menu. Even when I try to drag and drop notes from one folder to another it doesn't work. 

I am using Windows 10. 
Standard Notes version 2.3.4

**Comment by atmoio** [comment]
Hey @makarandh, this is a limitation for now. There's no drag and drop support other than to nest one folder into another. I think we're likely to add this at some point in the future.
